data/editor.config

```diff
@@ -940,7 +973,7 @@
       </type>
       <type id="tourism-picnic_site" />
       <type id="leisure-picnic_table" />
-      <type id="amenity-brothel" can_add="no">
```

Comment:
Please, don't. It may trigger incompatibility with "app for kids" in app stores. It is also a popular vandalism tool.

Reply:
We do display such POIs on the map (and e.g. stipclubs, gambling, sex shops...) for quite awhile already, and they're searchable.
How their addability would change anything?

Reply:
Yes. Please remove this line.

Reply:
Could you please elaborate why displaying brothels on the map is OK, but adding them to OSM is not?

Reply:
There are several reasons, I've already listed a few above.

Reply:
So why those reasons don't apply to displaying brothel POIs, but apply to adding them to OSM?
And why they don't apply to sex shops and strip clubs?

Reply:
I'd like to follow a consistent approach here.

Reply:
I was not aware of any strip clubs in the editor.

Drawing on the map is ok. Showing them in the list in all countries for all users, including children, is not ok.

Reply:
Ok, I see your point.!